CoroutineTemplate/app/src/main/java/co/nimblehq/coroutine/dispatcher/DispatchersProviderImpl.kt

```diff
@@ -0,0 +1,14 @@
+package co.nimblehq.coroutine.dispatcher
+
+import kotlinx.coroutines.CoroutineDispatcher
+import kotlinx.coroutines.Dispatchers
+import kotlinx.coroutines.MainCoroutineDispatcher
+
+class DispatchersProviderImpl : DispatchersProvider {
+
+    override fun getMain(): MainCoroutineDispatcher = Dispatchers.Main
+
+    override fun getIO(): CoroutineDispatcher = Dispatchers.IO
+
+    override fun getDefault(): CoroutineDispatcher = Dispatchers.Default
```

Comment:
the order is a bit different from the `DispatchersProvider` interface: IO > Main > Default vs Main > IO > Default;

Reply:
@sleepylee Oh, I actually didn't care about the order, but anyway let me update to make them more consistent.